internal/rest/avi_datascript.go

```diff
@@ -159,7 +159,9 @@ func (rest *RestOperations) AviDSCacheAdd(rest_op *utils.RestOp, vsKey avicache.
 
 		// Datascript should not have a checksum
 		checksum := lib.DSChecksum(ds_cache_obj.PoolGroups, nil, false)
-		checksum = utils.Hash(fmt.Sprint(checksum) + utils.HTTP_DS_SCRIPT_MODIFIED)
+		for _, pgName := range ds_cache_obj.PoolGroups {
```

Comment:
`DS` will point to single `PG`. Do we need it?

Reply:
Some UTs of panicked when I accessed the PoolGroup with index 0.